http-api/src/main/java/net/runelite/http/api/hiscore/Skill.java

```diff
@@ -49,7 +49,7 @@ public int getLevel()
 
 	public long getExperience()
 	{
-		return experience;
+		return experience < 0 ? 0 : experience;
```

Comment:
This class is really just a POJO so the handling of -1 should probably not be here.